moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
There are lots of instances of this in moveit that had it wrong, fixed in https://github.com/ros-planning/moveit/pull/254